scarb/tests/e2e/subcommand.rs

```diff
@@ -40,6 +40,42 @@ fn subcommand() {
         .stdout_eq("Hello beautiful world\n");
 }
 
+#[test]
+#[cfg(unix)]
+fn env_variables_are_passed() {
+    let t = TempDir::new().unwrap();
+    write_script(
+        "env",
+        indoc!(
+            r#"
+            #!/bin/bash
+            required=(
+                PATH
+                SCARB
+                SCARB_CACHE
+                SCARB_CONFIG
+                SCARB_LOG
+                SCARB_MANIFEST_DIR
+            )
+            for v in "${required[@]}"; do
+                if test -z "${!v}"
+                then
+                    echo "Variable $v is not set!"
+                    exit 1
+                fi
+            done
+            "#
+        ),
```

Comment:
nitpick, people usually use `{}` with `indoc!`, see examples here: https://docs.rs/indoc/latest/indoc/

```suggestion
        indoc!{r#"
            #!/bin/bash
            required=(
                PATH
                SCARB
                SCARB_CACHE
                SCARB_CONFIG
                SCARB_LOG
                SCARB_MANIFEST_DIR
            )
            for v in "${required[@]}"; do
                if test -z "${!v}"
                then
                    echo "Variable $v is not set!"
                    exit 1
                fi
            done
        "#},
```